e2e-tests/init-deploy/00-deploy-operator.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    . ../functions
+    set -o errexit
+    set -o xtrace
+
+    kubectl -n ${NAMESPACE} apply -f ${src_dir}/deploy/crd.yaml
+    kubectl -n ${NAMESPACE} apply -f ${src_dir}/deploy/rbac.yaml
+    yq --doc 1 w ${src_dir}/deploy/operator.yaml 'spec.template.spec.containers[0].image' "${IMAGE}" | kubectl -n ${NAMESPACE} apply -f -
```

Comment:
let's switch to yq 4

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/21/commits/ce9dbd37615ce61ca476bb38d31829eadc785ddd